services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowReroutePayload.java

```diff
@@ -52,4 +53,14 @@
 
     @JsonProperty("rerouted")
     private boolean rerouted;
+
+    @JsonCreator
+    public FlowReroutePayload(
+            @JsonProperty(value = Utils.FLOW_ID) String id,
+            @JsonProperty(value = Utils.FLOW_PATH) PathInfoData path,
+            @JsonProperty(value = "rerouted") boolean rerouted) {
```

Comment:
I discussed with @sergii-iakovenko and it seems that the constants approach is being deprecated as for now. This will require some refactoring across the project to use the same style everywhere, but it is not part of this task.

Reply:
Sure. It's not critical.